src/value.rs

```diff
@@ -817,16 +818,16 @@ impl SetValueOptional for str {
     }
 }
 
-impl<'a> FromValueOptional<'a> for Vec<String> {
+impl<'a> FromValueOptional<'a> for Vec<GString> {
     unsafe fn from_value_optional(value: &'a Value) -> Option<Self> {
-        Some(<Vec<String> as FromValue>::from_value(value))
+        Some(<Vec<GString> as FromValue>::from_value(value))
     }
 }
 
-impl<'a> FromValue<'a> for Vec<String> {
+impl<'a> FromValue<'a> for Vec<GString> {
     unsafe fn from_value(value: &'a Value) -> Self {
         let ptr = gobject_ffi::g_value_get_boxed(value.to_glib_none().0) as *const *const c_char;
-        FromGlibPtrContainer::from_glib_none(ptr)
+        GString::from_ptr_container(ptr)
```

Comment:
There really needed to remove implementations for `Vec<String>` ?

Reply:
I can leave it :)